src/main/java/scala_maven/ExtendedScalaContinuousCompileMojo.java

```diff
@@ -62,7 +62,7 @@ public File getTestAnalysisCacheFile() throws Exception {
     // Latest 3.3.1 fails to find java home for Spark (apparently), so we use the logic in 3.2.2
     public File getJavaHome() throws Exception {
         String _javaExec = null;
-        Toolchain toolchain = toolchainManager.getToolchainFromBuildContext("jdk", session);
```

Comment:
`toolchainManager` was made private in the new version, but instead there is a `getToolchain` that does exactly what we were doing here.